src/symbol/get_anchors.js

```diff
@@ -59,12 +59,9 @@ function getCenterAnchor(line: Array<Point>,
 
             const anchor = new Anchor(x, y, b.angleTo(a), i);
             anchor._round();
-            if (angleWindowSize &&
-                !checkMaxAngle(line, anchor, labelLength, angleWindowSize, maxAngle)) {
-                return;
+            if (!angleWindowSize || checkMaxAngle(line, anchor, labelLength, angleWindowSize, maxAngle)) {
+                return anchor;
             }
```

Comment:
Should this still always return once it reaches the middle segment?

Reply:
Oh yeah, good point. I think without that return it's going to just waste cycles iterating to the end of the line.

Reply:
Fixed.